conf/db/migration/metrics_portal_ddl/V18__grafana_report_source.sql

```diff
@@ -0,0 +1,19 @@
+/**
```

Comment:
I think this should be `/*` not `/**`.

Reply:
Hmm, all the other `.sql` files in this directory use `/**`.